src/station_cmd.cpp

```diff
@@ -4486,16 +4498,8 @@ CommandCost ClearTile_Station(TileIndex tile, DoCommandFlag flags)
 		case STATION_RAIL:     return RemoveRailStation(tile, flags);
 		case STATION_WAYPOINT: return RemoveRailWaypoint(tile, flags);
 		case STATION_AIRPORT:  return RemoveAirport(tile, flags);
-		case STATION_TRUCK:
-			if (IsDriveThroughStopTile(tile) && !CanRemoveRoadWithStop(tile, flags)) {
-				return_cmd_error(STR_ERROR_MUST_DEMOLISH_TRUCK_STATION_FIRST);
-			}
-			return RemoveRoadStop(tile, flags);
-		case STATION_BUS:
-			if (IsDriveThroughStopTile(tile) && !CanRemoveRoadWithStop(tile, flags)) {
-				return_cmd_error(STR_ERROR_MUST_DEMOLISH_BUS_STATION_FIRST);
-			}
-			return RemoveRoadStop(tile, flags);
+		case STATION_TRUCK:    FALLTHROUGH;
```

Comment:
```suggestion
		case STATION_TRUCK:    [[fallthrough]];
```
or maybe just nothing in this case (we removed `FALLTHROUGH` some time ago.

Reply:
CODINGSTYLE.md does not reflect that...